srcpkgs/stig/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'stig'
+pkgname=stig
+version=0.10.0a
+revision=1
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+depends="python3 python3-aiohttp python3-blinker python3-natsort python-urwid
```

Comment:
python3-urwid you mean ?

doesn't make sense for a python3 package to depend on python2

Reply:
Yes, apologies.